src/collective/collectionfilter/filteritems.py

```diff
@@ -81,6 +81,9 @@ def get_filter_items(
 
     # Recursively transform all to unicode
     request_params = safe_decode(request_params)
+    # Things break if sort_order is not a str
+    if "sort_order" in request_params:
```

Comment:
I'd recommend to do this only in PY2 with:
```
if six.PY2 and "sort_order" in request_params:
```

Reply:
ok. but its the same either way. in py3 its a str also.

Reply:
AFAIK in PY3 everything should be a str (which is unicode in PY2)